### ISSUE TITLE
Create a new Optimizer: Train Mask-RCNN with SWA

### ISSUE THREAD

**Issue by yc0619** [open-mmlab/mmdetection#4793]
Hi,

according to the issue #4698, #3086 is the tutor 5 not the right way to customize the optimizer.

Could someone help how can i train Mask rcnn with SWA from torch.optim?

Thx.

**Comment by yc0619** [comment]
I have seen that the new version mmdetection has support the swa with mask rcnn. There's no need to create a new optimizer.

**Comment by Bo396543018** [comment]
@ yc0619 Hi, Could you tell me where is the config for the swa with mask rcnn?

**Comment by heartInsert** [comment]
where is  it  ?  Friend ,I'm  looking  for  it .
